Extension/PesterTask/PesterV9/Pester.ps1

```diff
@@ -47,6 +47,10 @@ if ($TargetPesterVersion -match '^5') {
     Write-Host "##vso[task.logissue type=error]This version of the task does not support Pester V5, please use task version 10 or higher."
     exit 1
 }
+if ($TargetPesterVersion -match 'latest') {
+    Write-Host "##vso[task.logissue type=warning]Task version not properly set so defaulting to enforcing it to be the latest possible v4 that we will support in this version of the task - v4.99.99"
+    $TargetPesterVersion = '4.99.99'
```

Comment:
Don't need this change to the target pester version, the import script handles "latest" fine.